cime_config/tests.py

```diff
@@ -459,6 +459,7 @@
         "time"  : "01:00:00",
         "tests" : (
             "ERP_D_Ln9.ne4_ne4.F2010-SCREAMv1",
+            "ERS_Ld5.ne4_ne4.F2010-SCREAMv1",
```

Comment:
Why adding a new ERS test?

Reply:
I instead replaced the ERP_D test with an Lh4 instead of Ln9.  The purpose is to confirm that we have BFB restarts when the frequency is not number of steps.